ai_bridge/models/ai_bridge_execution.py

```diff
@@ -0,0 +1,82 @@
+# Copyright 2025 Dixmit
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+import requests
+
+from odoo import _, api, fields, models
+
+
+class AiBridgeExecution(models.Model):
+
+    _name = "ai.bridge.execution"
+    _description = "Ai Execution"  # TODO
+
+    ai_bridge_id = fields.Many2one(
+        "ai.bridge",
+        required=True,
+        ondelete="cascade",
+    )
+    res_id = fields.Integer(required=True)
+    model_id = fields.Many2one(
+        "ir.model",
+        required=True,
+        ondelete="cascade",
+    )
+    payload = fields.Json(readonly=True)
+    result = fields.Text(readonly=True)
+    company_id = fields.Many2one(
+        "res.company",
+        compute="_compute_company_id",
+        store=True,
+        readonly=True,
+    )
+
+    @api.depends("ai_bridge_id")
+    def _compute_company_id(self):
+        for record in self:
+            record.company_id = record.ai_bridge_id.company_id
+
+    def _execute(self, **kwargs):
+        self.ensure_one()
+        payload = self.ai_bridge_id._prepare_payload(
+            self.env[self.model_id.model].browse(self.res_id), **kwargs
+        )
+        response = requests.post(
+            self.ai_bridge_id.url,
+            json=payload,
+            auth=self._get_auth(),
+            headers=self._get_headers(),
+            timeout=30,  # Default timeout, can be overridden by _execute_kwargs
+            **self._execute_kwargs(**kwargs),
+        )
+        response.raise_for_status()
+        self.result = response.content()
```

Comment:
This error would have to be controlled:
File "/opt/odoo/auto/addons/ai_bridge/models/ai_bridge_execution.py", line 53, in _execute
odoo-1        |     self.result = response.content()
odoo-1        | TypeError: 'bytes' object is not callable